libs/subliminal_patch/providers/subssabbz.py

```diff
@@ -144,8 +154,17 @@ def query(self, language, video):
                     link = element.get('href')
                     element = row.find('a', href = re.compile(r'.*showuser=.*'))
                     uploader = element.get_text() if element else None
-                    logger.info('Found subtitle link %r', link)
-                    sub = self.download_archive_and_add_subtitle_files(link, language, video)
+                    cells = row.findChildren('td')
+
+                    sub_disks = int(cells[6].getText()) if cells else 1
+                    if sub_disks != 1:
+                        logger.info('Multi-disk subtitles (unsupported). Skipping %r', link)
+                        continue
+
+                    sub_fps = float(cells[7].getText()) if cells else 0
+                    logger.info('Found subtitle link %r, fps %s, video %s', link, sub_fps, video.fps)
+
+                    sub = self.download_archive_and_add_subtitle_files(link, language, video, sub_fps)
                     for s in sub: 
                         s.uploader = uploader
```

Comment:
It's better is you set fps as `s.fps = sub_fps`. Not passing it as parameter to download_archive_and_add_subtitle_files() will reduce code modification, so PR will become simpler.

Reply:
The patch uses the same principles as with the Opensubtitles provider codebase - see https://github.com/morpheus65535/bazarr/blob/development/libs/subliminal_patch/providers/opensubtitles.py#L325

That said, I'm not a fan of your "patchy" inline `s.uploader = uploader` assignment either. Obscured inline assignment in the middle of the code is significantly less readable than having `self.uploader` assignment in `__init__`.

Reply:
I also prefer having class attribute but I can live with both. :-)